go.mod

```diff
@@ -98,6 +98,7 @@ require (
 	gopkg.in/square/go-jose.v2 v2.5.1
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	gotest.tools v2.2.0+incompatible
```

Comment:
Is this intended to be added? Not sure what is this about.

Reply:
I just run `go mod tidy` and this package was updated.